release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1145,6 +1145,10 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-olm-bug-fixes"]
 ==== Operator Lifecycle Manager (OLM)
 
+* The status of an Operator’s custom resource (CR) includes a list of components owned by the Operator. This list is ordered by group/version/kind (GVK), but the order of objects with the same GVK might change. If an Operator owns many components with the same GVK, it can cause Operator Lifecycle Manager (OLM) to continuously update the status of the Operator CR, because the order of its components has changed. This bug fix updates OLM so that the order of an Operator's component references is deterministic. As a result, OLM no longer attempts to update the CR repeatedly when the list of components remains constant. (link:https://issues.redhat.com/browse/OCPBUGS-2556[*OCPBUGS-2556*])
+
+* Operator Lifecycle Manager (OLM) manages a set of `CatalogSource` objects from which Operators can be searched for and installed from. These catalog sources are the default sources for this action and are managed by Red Hat. However, it was possible to change these default catalog sources in a way that the OLM system would not notice. Modifying a default catalog source in a way that rendered it inoperable could cause cascading issues through OLM that could prevent a user from installing new or upgrading existing Operators on their cluster. This bug fix updates the `catalog-operator` runtime, which manages the default catalog sources, to be made aware of other changes to the `CatalogSource` spec. As a result, when a change is made to a default catalog source, OLM detects the change and resets it back to default. (link:https://issues.redhat.com/browse/OCPBUGS-5466[*OCPBUGS-5466*])
```

Comment:
I think might might work better here for clarity since "could" is mentioned already in the sentence.
```suggestion
* Operator Lifecycle Manager (OLM) manages a set of `CatalogSource` objects from which Operators can be searched for and installed from. These catalog sources are the default sources for this action and are managed by Red Hat. However, it was possible to change these default catalog sources in a way that the OLM system would not notice. Modifying a default catalog source in a way that rendered it inoperable could cause cascading issues through OLM that might prevent a user from installing new or upgrading existing Operators on their cluster. This bug fix updates the `catalog-operator` runtime, which manages the default catalog sources, to be made aware of other changes to the `CatalogSource` spec. As a result, when a change is made to a default catalog source, OLM detects the change and resets it back to default. (link:https://issues.redhat.com/browse/OCPBUGS-5466[*OCPBUGS-5466*])
```